CRM/Prospect/Setup/CreateProspectWorkflowCaseType.php

```diff
@@ -43,7 +81,7 @@ public function apply() {
    *   The status definitions
    */
   private function getStatusDefinition() {
-    $workflowStatuses = ProspectWorkflowCaseStatusesSetup::getStatuses();
+    $workflowStatuses = CRM_Prospect_Setup_CreateProspectWorkflowCaseStatuses::getStatuses();
```

Comment:
Any reason why you changed this to use the full name instead of the alias?

Reply:
well the linters asked for not using use statement for non namespaced classes

Reply:
Then we need to fix this as we should be able to use the `use` statement since we do the same in civicase and civiawwards extensions. You can check the particular rule we need to exclude and do that in the `phpcs-ruleset.xml`. Also check this file in other case suite extensions for reference.

Better still, I think we should switch to the civicrm coder rather than using the drupal coder, this will be better as the coder is suited for civicrm code development and we won't really need to exclude rules.  See this: https://github.com/compucorp/uk.co.compucorp.civicrmnewrelic/blob/master/.github/workflows/linters.yml#L21 to check the coder I was referring to.